src/gbd_mapping_generator/covariate_builder.py

```diff
@@ -18,8 +18,8 @@ def get_base_types():
                       ('low_value_exists', 'bool'),
```

Comment:
it won't let me comment on the line above but I think it should be 'data_exists'

Reply:
I take it all back. Michelle is right that data is plural